components/typography/config.less

```diff
@@ -1,12 +1,14 @@
 @line-height-base: 1.4;
 
-@font-size-base: 14px;
-@font-size-xsmall: 11px;
-@font-size-small: 12px;
-@font-size-large: 16px;
+@font-size-base: 15px;
+@font-size-xsmall: 12px;
+@font-size-small: 13px;
+@font-size-large: 17px;
+
 // Open Sans cap height
 @cap-height: 0.7727272727;
 
-@font-family-sans-serif: "open-sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
-@font-family-serif: "ff-tisa-sans-web-pro", serif;
+// Todo : Better name for font family
+@font-family-sans-serif: "proxima nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
+@font-family-serif: "effra", sans-serif;
```

Comment:
Can you add another variable that better describes the effra font stack and alias it to `@font-family-serif`?

Something like

```
@font-family-secondary: @font-family-serif;
```